Common/FileUtil.cpp

```diff
@@ -0,0 +1,39 @@
+﻿/* Copyright (C) 2015 Rainmeter Project Developers
+*
+* This Source Code Form is subject to the terms of the GNU General Public
+* License; either version 2 of the License, or (at your option) any later
+* version. If a copy of the GPL was not distributed with this file, You can
+* obtain one at <https://www.gnu.org/licenses/gpl-2.0.html>. */
+
+#include "StdAfx.h"
+#include "FileUtil.h"
+
+namespace FileUtil
+{
```

Comment:
Can you change this to `namespace FileUtil {` and remove the indentation for the namespace block?